app/components/List/Item.tsx

```diff
@@ -63,13 +63,13 @@ const ListItem = (
   }
 
   return (
-    <Wrapper $border={border} $small={small} {...rest}>
+    <Wrapper ref={ref} $border={border} $small={small} {...rest}>
       {content(false)}
     </Wrapper>
   );
 };
 
-const Wrapper = styled.div<{ $small?: boolean; $border?: boolean }>`
+const Wrapper = styled.a<{ $small?: boolean; $border?: boolean; to?: string }>`
```

Comment:
Should we just make this a `CompositeItem`? Would avoid needing to do it separately in every item that inherits from this.

Reply:
I played with a lot of permutations of this, including what you suggested, but I went with this way in order to support inactive items (which are removed from the composite keyboard navigation but not disabled)

If we make these composite items we can give them the disabled prop but it removes all of the interactivity including mouseovers of the Time component and such. If that's a tradeoff you're happy with, we can go that way